examples/dataproc/main.py

```diff
@@ -19,6 +19,7 @@
     packages=['click', 'selenium'],
     init=['gs://cpg-reference/hail_dataproc/install_common.sh'],
     job_name='example',
+    labels={'compute_category': 'dataproc', 'stage': 'example_usage'},
```

Comment:
Wouldn't we want to set the category automatically, as these are all Dataproc jobs? Don't we already know that from the SKU?

Reply:
So my line of thinking was that if we wanted to be able to break down particular costs associated with seqr (for example, annotation) then a stage=annotation label here would be useful. I was considering a scenario where `hail_dataproc_function` is used in different contexts, and therefore different breakdowns may be useful. In passing in these labels, and defining them in for example `hail-elasticsearch-pipelines` as opposed to here we can breakdown the costs with whatever level of granularity we like, and change flexibly according to our use case. Perhaps I can set this variable to be optional and automatically set these categories if the user doesn't provide them? Or is there a better way to achieve this?

Reply:
Completely agree that we want to allow additional, optional labels. But isn't the category here fixed / redundant?

Reply:
Oh, this is an example script that was created elsewhere, and because the labels were set as compulsory in `dataproc.py` I had to pass in values. This doesn't change the functionality of the `dataproc.py` script. Happy to switch this out to an optional variable though in `dataproc.py` and then this `examples/dataproc/main.py` can remain as it was before.

Reply:
Yes, I think it would be good to keep the example as before, for simplicity.